src/components/AppComponents/AppFooter/index.tsx

```diff
@@ -1,85 +1,147 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { useTranslation } from 'react-i18next';
 import { Link } from 'react-router-dom';
+import AppContextEducation from './../../../samples/EducationStart/reuseables/AppContextEducation'; // TODO: Once this code exposed to common folder, we will refer AppContext from reuseable components
+import { useLocation } from 'react-router-dom';
 
 export default function AppFooter() {
   const { t } = useTranslation();
+  const location = useLocation();
+  const { serviceParam, serviceName, appNameHeader } = useContext(AppContextEducation);
+
+  const searchParams = new URLSearchParams(location.search);
+
+  // Default for all services
+  const serviceParamUpdate = serviceParam || searchParams.get('serviceParam') || '463';
+  const appNameUpdate =  appNameHeader || searchParams.get('appname') || 'CLAIM_CHILD_BENEFIT';
+  const serviceNamePageTitle = serviceName || searchParams.get('serviceNamePageTitle') || 'CLAIM_CHILD_BENEFIT';
```

Comment:
Instead of editing the footer, could we not alter the header element as that is where the issue is?

So in React context have a variable that is the appname i.e. Child claim, HICBC etc. And at the start of each journey we set that global state in the context. So if we come from any page in a flow you will see the right header in the cookies page.

Then in the AppHeader we check that state and display the right header title.